indexes/private.js

```diff
@@ -39,14 +37,16 @@ module.exports = function (dir, keys) {
 
         cb(null, { seq, arr: fic.uncompress(body) })
       })
-      .catch((err) => cb(err))
+      .catch(cb)
   }
 
   function loadIndexes(cb) {
     load(encryptedFile, (err, data) => {
       if (err) {
         latestSeq.set(-1)
-        return cb(err)
+        if (err.code === 'ENOENT') cb()
+        else cb(err)
+        return
```

Comment:
We care about errors, unless the index does not exist previously (which is a common case when we're starting up the database)